frontend/src/components/InputDialogs/InputDialogs.tsx

```diff
@@ -389,14 +389,19 @@ const InputDialogs = () => {
             />
           </InputWrapper>
           <InputWrapper>
-            <Input
+            <select
```

Comment:
```suggestion
            <Input type="select"
```
Use this to use the Goober styling